proposal.rst

```diff
@@ -260,18 +347,60 @@ Project Timeline
 .. and/or exams in this time, so we certainly aren't expecting full time work
 .. on your project, but you should aim to complete preliminary work such that
 .. you can actually start coding at the start of the coding period.
-.. 
+..
 .. The coding period is broken into three blocks of about 4 weeks each, with
 .. an evaluation after each block.  The evaluations are to help keep you on
 .. track, and consist of brief evaluation forms sent to GSoC by both the
 .. student and the mentor, and a chance to explicitly review how your project
 .. is going with Xapian mentors.
-.. 
+..
 .. If you will have other commitments during the project time (for example,
 .. any university classes or exams, vacations, etc), make sure you include them
 .. in your project timeline.
 
-FILLME
+
+
+**Community Bonding Period: (May 4 - June 1) Module 1,2**
+
+  * Get to know the community and interact with its members over the entire bonding period.
+  * Discuss and find a minor issue directly related to the project that needs to be fixed. (2 days)
+  * Identify a solution to the issue and discuss with members. (2 days)
+  * Write code to fix the issue, write relevant tests and documentation as necessary. Familiarize with the development workflow. (3 days)
```

Comment:
> You've already worked on a small PR. What else do you think you need to learn about the workflow? It's helpful to be specific, both because that will help us support you but also because it will help you think about the work you'll need to do.

Although I have worked on a small PR to get familiar with the workflow, this was tangentially related to the project. I read [https://xapian-developer-guide.readthedocs.io/en/latest/tests/index.html](url) , I do have an idea about how to add libraries but not specifically for testing. So I will require getting familiar with it.